selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -290,12 +296,15 @@ public void clickOnStackButton() {
 
   public void clickOnWorkspaceStopStartButton(String workspaceName) {
     String buttonXpath = String.format(WORKSPACE_ITEM_STOP_START_WORKSPACE_BUTTON, workspaceName);
+    waitProgressBarInvisibility();
```

Comment:
Do we really need to wait for progress bar invisibility at the start of method? 
It is expected that workspace is completely ready to perform an operation, isn't it?

Reply:
No, by that case we ensuring that loader is not displayed because if it is present (because of some previous actions) click event may be lost.

Reply:
I see. But the method propose is just clicking on workspace stop/start button. It doesn't declare waiting on something before clicking. So, IMHO, it's client responsibility to ensure that there is nothing blocks clicking on the button.